OpenWeather/Utilities/Operators/==.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  OpenWeather template generated by OpenBytes on 16/05/2023.
+//
+// Created by Ahmed Shendy.
+//  ==.swift
```

Comment:
Maybe change the file name to not use symbols.. Only symbol I use normally in file names is `+`.

Maybe this could be `LocationEquating` or something similar.